rero_ils/modules/users/jsonschemas/users/user-v0.0.1.json

```diff
@@ -65,6 +70,14 @@
         }
       }
     },
+    "gender": {
+      "title": "Gender",
+      "enum": [
+        "male",
+        "female",
+        "other"
```

Comment:
I think that something like "inapplicable" should be in the list, for institution.

Reply:
Please check this with the PO.

Reply:
@jma Checked. As the field is not required, nothing is necessary for institutional account. Sorry.

Reply:
@iGormilhit thanks for you review and for the PO checkings.